src/content/docs/new-relic-one/use-new-relic-one/ui-data/maps-in-context.mdx

```diff
@@ -32,9 +34,9 @@ The typical map view shows you entities, represented as hexagons, and their rela
         Large, hollow hexagon
       </td>
       <td>
-        Primary entities that are the focus of the current map. 
+        The primary entities that are the focus of the current map. 
         
-        For example, the entities involved in an AIOps Issue or a Distributed Trace. If the entity is monitored by a New Relic agent, its language or type will be indicated by the icon inside the hexagon.
+        For example, for a distributed trace or [Applied Intelligence](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence/) issue, those would be the entities directly involved. If the entity is monitored by a New Relic agent, its language or type is indicated by the icon inside the hexagon.
```

Comment:
Since you use `Applied Intelligence` as an example throughout this doc, I would recommend linking to the `Intro to Applied Intelligence` doc earlier in this doc.